server/viame_server/utils.py

```diff
@@ -2,10 +2,11 @@
 from girder.models.folder import Folder
 
 
-validImageFormats = {"png", "jpg", "jpeg"}
+validImageFormats = {"png", "jpg", "jpeg", "tif", "tiff", "sgi", ".bmp", ".pgm"}
```

Comment:
I'm guessing the mimeType info includes thhe `.` as part of the extension.  Just wondering why `.bmp` and `.pgm` have the `.`?  We use the full extension on the client side (`.png, .jpg`) and here we exclude.  I like the idea of the backend giving an endpoint of the available image/video types and utilizing that.  Just wondering if we should go or the other.

Reply:
The inclusion of periods is a mistake, I'll fix it.



> Just wondering if we should go or the other.

What do you mean by this?

Reply:
@BryonLewis ptal at 8b5ffdc

Reply:
> The inclusion of periods is a mistake, I'll fix it.
> 
> > Just wondering if we should go or the other.
> 
> What do you mean by this?

my mistake, "one way or the other"